### PR TITLE
ensure controlled conjurations aren't considered in narratives

### PR DESCRIPTION
**Pull request by ThyWoof**

### CODE REVIEW

File: SolastaCommunityExpansion/Patches/SrdAndHouseRules/FullyControlConjurations/NarrativeDirectionManagerPatcher.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+namespace SolastaCommunityExpansion.Patches.SrdAndHouseRules.FullyControlConjurations
+{
+    [HarmonyPatch(typeof(NarrativeDirectionManager), "PrepareDialogSequence")]
+    [SuppressMessage("Minor Code Smell", "S101:Types should be named in PascalCase", Justification = "Patch")]
+    internal static class NarrativeDirectionManager_StartDialogSequence_Patch
+    {
+        internal static void Prefix(List<GameLocationCharacter> involvedGameCharacters)
+        {
+            if (!Main.Settings.FullyControlConjurations)
```
[the remark this answers is not in the document]
Comment (PhilPJL) [reply SolastaCommunityExpansion/Patches/SrdAndHouseRules/FullyControlConjurations/NarrativeDirectionManagerPatcher.cs:13]: It's unlikely to be an issue either way, but I don't have enough knowledge to say for sure.

**Review by PhilPJL — APPROVED** [review]
I'll let you decide on taking the check out or not.

File: SolastaCommunityExpansion/Models/ConjurationsContext.cs [hunk 2]
```diff
@@ -9,42 +10,50 @@ namespace SolastaCommunityExpansion.Models
...
+            FeyGiantApe, // CR 6
+            FeyGiant_Eagle, // CR 5
+            FeyBear, // CR 4
+            Green_Hag, // CR 3
+            FeyWolf, // CR 2
+            FeyDriad, // CR 1
+        };
+
+        /// <summary>
+        /// Allow conjurations to fully controlled party members instead of AI controlled.
```
Comment (PhilPJL) [inline SolastaCommunityExpansion/Models/ConjurationsContext.cs:45]: Allow conjurations to be fully controlled by party members instead of AI controlled.

Comment (ThyWoof) [reply SolastaCommunityExpansion/Models/ConjurationsContext.cs:45]: I’ll leave it as this would be very rare and better keep the setting protection

**Pull request merged**